addons/web/static/src/xml/file_upload_progress_bar.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+<t t-name="web.ProgressCard">
+    <t t-if="widget.viewType === 'kanban'">
+        <div class="o_kanban_progress_card o_kanban_record o_kanban_attachment">
+            <div class="o_kanban_image">
+                <div class="o_kanban_image_wrapper">
+                    <div class="o_image o_image_thumbnail" t-att-data-mimetype="widget.type"/>
+                </div>
+            </div>
+            <div class="o_kanban_details">
+                <div class="o_kanban_details_wrapper">
+                    <div t-att-title="widget.title" class="o_kanban_record_title">
```

Comment:
this usually applies to all templates: whenever there is a `title`, an `aria-label` with similar value is expected